docs/reference/mobile/android/setup.md

```diff
@@ -2,36 +2,88 @@
 
 ## Set up your Unity Editor
 
 Most of your interactions with the GDK happen inside your Unity Editor. To get started:
 
-1. Open your Unity Editor.
-1. It should automatically detect your project. If it doesn't, select **Open**, navigate to `<path-to-your-project>/workers/unity` and select **Open**.<br>
-If you don’t have a SpatialOS Unity project you can use the [FPS Starter Project]({{urlRoot}}/projects/fps/get-started/get-started) or the [Blank Starter Project]({{urlRoot}}/projects/blank/overview) to get started.
-1. In your Unity Editor, go to **File** > **Build Settings**. Select **Android** and then **Switch Platform**.
-1. Still in your Unity Editor, add the file paths Unity needs via the **External Tools** window. To do this:
-    * on Windows, go to **Edit** > **Preferences**
-    * on MacOS, go to **Unity** > **Preferences**.
-
-    In the **Android** section of the **External Tools** window,  input the paths to the SDK, JDK and NDK. The easiest way to ensure that Unity reads the file path correctly is to use the browse option:
-
-| Field | How to find the path |
-|-------|------|
-| SDK  |  You can find the SDK location by opening Android Studio and selecting **Configure** then **SDK Manager**. |
-| JDK  |  **Windows:** default path is `C:/Program Files/Java` <br/>**Mac:** run `which java` to retrieve the path. |
-| NDK  |  (Optional) Use the directory path that you noted down earlier when extracting the Android NDK.|
+1. Open your SpatialOS project inside the Unity Editor. It is located inside `<path-to-your-project>/workers/unity`.
+
+    > If you don’t have a SpatialOS Unity project, you can use the [FPS Starter Project]({{urlRoot}}/projects/fps/get-started/get-started) or the [Blank Starter Project]({{urlRoot}}/projects/blank/overview) to get started.
+
+1. In your Unity Editor, go to the **External Tools** window. To do this:
+    * on Windows, go to **Edit** > **Preferences** > **External Tools**
+    * on MacOS, go to **Unity** > **Preferences** > **External Tools**
+
+1. In the **Android** section of the **External Tools** window, tick the **Use embedded JDK** checkbox to use the JDK that Unity provides. 
+1. Input the following paths into the **Android Section**:
+
+	| Field | How to find the path |
+	|-------|------|
+	| SDK  |  You can find the SDK location by opening Android Studio and selecting **Configure** then **SDK Manager**. |
+	| NDK  |  (Optional) Use the directory path that you noted down earlier when extracting the Android NDK.|
+
+1. (Optional, if you want to use Unity Remote) Go to **Edit** > **Project Settings** > **Editor** to bring up the **Editor Settings** window and navigate to the **Unity Remote** section. Set the **Device** setting to **Any Android Device**.
+
+## Set up your project
+
+If you are using one of our [Starter Projects]({{urlRoot}}/reference/glossary#starter-project), you can skip the **Create a mobile connector script** section, as you already have one in your project.
```

Comment:
Maybe `>` this to highlight it more?

Reply:
but then we have a callout and a collapsable section right after another?